hypothesis-python/src/hypothesis/_strategies.py

```diff
@@ -1305,6 +1305,25 @@ def from_type(thing):
        other elements in the lookup.
     4. Finally, if ``thing`` has type annotations for all required arguments,
        it is resolved via :func:`~hypothesis.strategies.builds`.
+
+    There is a valuable recipe for leveraging ``from_type`` to generate
+    "everything except" values from a specified type. I.e.
+
+    .. code-block:: pycon
+
+        >>> def everything_except(excluded_types):
+        ...     return (
+        ...         # draws from the list of registered builtin types
+        ...         from_type(type)
+        ...         # maps the drawn type to its associated strategy
+        ...         .flatmap(from_type)
+        ...         .filter(lambda x: not isinstance(x, excluded_types))
+        ...     )
+
+     For example, ``everything_except(int)`` is a strategy that will generate
+     values from all of the strategies registered with the builtin types other
```

Comment:
I think we can be clearer here:

> ``everything_except(int)`` returns a strategy that can generate anything that `from_type()` can ever generate, except for instances of :class:`python:int`, including instances of types added via :func:`~hypothesis.strategies.register_type_strategy`.  This is useful when writing tests which check that invalid input is rejected in a certain way.

Reply:
> including instances of types added via :func:~hypothesis.strategies.register_type_strategy

Actually, I had a question about this. It doesn't seem like `from_type(type)` gets updated with registered types:

```python
class C: pass
register_type_strategy(C, none())

# this test passes
@given(from_type(type))
def test_custom_class(x):
    assert x is not C
```

And indeed, inspecting the list of types returned by `st.from_type(type)` suggests that `C` is not included.

Reply:
You're right!  That "including" should be "excluding" I guess...